stablehlo/reference/Ops.cpp

```diff
@@ -224,6 +225,47 @@ Tensor evalTransposeOp(const Tensor &operand, ArrayRef<int64_t> permutation,
   return result;
 }
 
+// ***********************************************************************
+// This is an simplified implementation of while op semantics. The
+// simplification is based on iterating the loop `dummy_limit` number of times
+// if the loop condition is `true`. The simplifiction is done as compare op is
+// not supported. To be corrected and improved as part of #967 and #992
+// respectively.
+// ***********************************************************************
```

Comment:
I think we should just hardcode support for ConvertOp directly in this function. That would probably take a similar amount of effort to implementing the notion of dummy_limit and explaining why we need it :)